qe_metrics/cli.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import Dict, Any
+
+import click
+
+from simple_logger.logger import get_logger
+
+from qe_metrics.libs.config import Config
+
+
+@click.group()
+@click.option(
+    "--config",
+    "-c",
+    default=os.environ.get("QE_METRICS_CONFIG", "config.yaml"),
+    help="Defines the path to the config file.",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--local-db",
+    is_flag=True,
+    help="Use a local SQLite database instead of a real database.",
```

Comment:
Maybe `Use a local SQLite database instead of the DB in the config file`?
Since `SQLite` is also real DB :)

Reply:
Also, do we need this? 
One can have two config files, one for prod and one for debug (local)

Reply:
I will update the help!

The only benefit I see to keeping it as a flag is that the user won't need to switch between config files for debugging, they can just add a flag and test their actual config using SQLite instead of the Postgres database. I am fine with moving this option to the config file if you think the benefit isn't enough to warrant the option. WDYT?

Reply:
Never mind, I think I will make it an option in the configuration file. I realized we need to separate the database and Jira credentials into a separate file from the configuration file per the requirements. Will update the code to reflect that, and will allow the option for a SQLite database in the file that defines the database credentials.